common/files_helper.js

```diff
@@ -0,0 +1,22 @@
+
+module.exports = {
```

Comment:
@borkarsaish65 why we need separate file for this? generic->messageconstants we already have . Please re use them. IF anything missing add there

Reply:
done